### PR TITLE
en: update manage-replication-tasks.md for 2.0

### PR DESCRIPTION
**Pull request by toutdesuite**
### What is changed, added or deleted? (Required)

update manage-replication-tasks.md for 2.0

### Which TiDB Data Migration (DM) version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v2.0 changes for now)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-dm/pull/35, https://github.com/pingcap/docs-dm/pull/49
- Other reference link(s):

### CODE REVIEW

**Comment by toutdesuite** [comment]
@TomShawn This PR is ready for review since https://github.com/pingcap/docs-dm/pull/49 has been merged. PTAL. Thanks~

File: en/manage-replication-tasks.md [hunk 1]
```diff
@@ -93,15 +117,76 @@ Available Commands:
...
+
+{{< copyable "" >}}
+
+```bash
+operate-source create ./source.toml
+```
+
+#### Flags description
+
++ `create`: Creates an upstream database source
```
Comment (TomShawn) [inline en/manage-replication-tasks.md:159]: Please end the sentence with a period.
```suggestion
+ `create`: Creates an upstream database source.
```

File: en/manage-replication-tasks.md [hunk 2]
```diff
@@ -93,15 +117,76 @@ Available Commands:
...
+
++ `update`: Updates an upstream database source
```
Comment (TomShawn) [inline en/manage-replication-tasks.md:161]: ditto

File: en/manage-replication-tasks.md [hunk 3]
```diff
@@ -93,15 +117,76 @@ Available Commands:
...
+
++ `stop`: Stops an upstream database source
```
Comment (TomShawn) [inline en/manage-replication-tasks.md:163]: ditto

File: en/manage-replication-tasks.md [hunk 4]
```diff
@@ -318,28 +398,16 @@ resume-task test
...
-             "meta": {
-                 "result": true,
-                 "worker": "172.16.30.16:8262",
-                 "msg": ""
-             },
-             "op": "Resume",
-             "logID": "3"
-         }
-     ]
+    "result": true,
```
Comment (TomShawn) [inline en/manage-replication-tasks.md:401]: It seems the indentation of Line 401- Line 403 is different from that of Line 400.

File: en/manage-replication-tasks.md [hunk 5]
```diff
@@ -482,25 +538,15 @@ update-task task_all_black.yaml
...
+    "msg": "",
+    "sources": [
+    ]
 }
 ```
 
 ## Manage DDL locks
 
-Currently, DDL lock related commands mainly include `show-ddl-locks`, `unlock-ddl-lock`, `break-ddl-lock`, etc. For more information on their functions, usages, and applicable scenarios, refer to [Handle Sharding DDL Locks Manually](feature-manually-handling-sharding-ddl-locks.md).
+Currently, DDL lock related commands mainly include `show-ddl-locks`, `unlock-ddl-lock`, etc. For more information on their functions, usages, and applicable scenarios, refer to [Handle Sharding DDL Locks Manually](feature-manually-handling-sharding-ddl-locks.md).
```
Comment (TomShawn) [inline en/manage-replication-tasks.md:549]:
```suggestion
Currently, DDL lock related commands mainly include `show-ddl-locks`, `unlock-ddl-lock`. For more information on their functions, usages, and applicable scenarios, refer to [Handle Sharding DDL Locks Manually](feature-manually-handling-sharding-ddl-locks.md).
```

**Comment by toutdesuite** [comment]
@TomShawn PTAL again. Thanks~

**Pull request merged**